editor/plugins/texture_editor_plugin.cpp

```diff
@@ -70,7 +70,48 @@ void TexturePreview::_update_metadata_label_text() {
 		format = texture->get_class();
 	}
 
-	metadata_label->set_text(vformat(String::utf8("%s×%s %s"), itos(texture->get_width()), itos(texture->get_height()), format));
+	const Ref<Image> image = texture->get_image();
+	if (image.is_valid()) {
+		const int mipmaps = image->get_mipmap_count();
+		uint64_t memory = image->get_width() * image->get_height() * Image::get_format_pixel_size(image->get_format());
```

Comment:
The intermediate values need to be converted as well. This will overflow before being cast to a ``uint64_t``